dev/tests/integration/testsuite/Magento/Catalog/Block/Product/ListTest.php

```diff
@@ -39,7 +43,7 @@ public function testGetLoadedProductCollection()
     {
         $this->_block->setShowRootCategory(true);
         $collection = $this->_block->getLoadedProductCollection();
-        $this->assertInstanceOf(\Magento\Catalog\Model\ResourceModel\Product\Collection::class, $collection);
+        $this->assertInstanceOf(\Magento\Eav\Model\Entity\Collection\AbstractCollection::class, $collection);
```

Comment:
Why did you change it?

Reply:
The reason i think we should check with corresponding type return 
https://github.com/magento/magento2/blob/7264026e018ce4baa84e944f03e749616d5471cd/app/code/Magento/Catalog/Block/Product/ListProduct.php#L141
This phpdoc return AbstractCollection abstract class

Reply:
That’s a bug in the php doc block that not so easy to fix due to SVC failures. Method getProductCollection should return product collection, so previously it was correct